pkgs/development/go-modules/generic/default.nix

```diff
@@ -20,6 +20,8 @@
 , vendorSha256 ? null
 # Whether to delete the vendor folder supplied with the source.
 , deleteVendor ? false
+# Whether to run the vend tool to regenerate the vendor directory.
+, runVend ? false
```

Comment:
Can you also add this to documentation in `languages-frameworks/go.xml`?

Reply:
Done.